cabal.project.stage1

```diff
@@ -81,6 +81,9 @@ package ghc-boot-th-next
 package hsc2hs
   flags: +in-ghc-tree
 
+package libffi-clib
+  ghc-options: -optc-Wno-error
+
 --
```

Comment:
We should document why this is needed. I think there were some warnings from the libffi-clib, which then became errors and just failed the build.